[W-15479032] Adding exchange mainFile property description note (#227)
mulesoft/docs-anypoint-cli: modules/ROOT/pages/_partials/exchange-assets.adoc

* `--properties='{"assetLink":"\http://api.com"}'`
* `--properties='{"mainFile":"main.raml","apiVersion":"v1","assetLink":"\http://api.com","contactEmail":"api@mulesoft.com","contactName":"Marc"}'`

> Note: the file of the specified "mainFile" property should be contained in the uploaded zip file. It should be on the root path, it can not be on a subfolder.

Reviewer: ```suggestion
NOTE: The file of the specified "mainFile" property must be in the uploaded zip file on the root path. The file cannot be in a subfolder.
```
